src/sonic-yang-models/yang-models/sonic-qos-pfc.yang

```diff
@@ -0,0 +1,82 @@
+module sonic-qos-pfc {
+    namespace "http://github.com/Azure/sonic-qos-pfc";
+    prefix sonic-qos-pfc;
+
+    yang-version 1.1;
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC QoS Priority-based Flow Control";
+
+    revision 2021-07-01 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-qos-pfc {
+
+        container PFC_WD {
+            list PFC_WD_LIST {
+                key "ifname";
+
+                leaf ifname {
+                    mandatory true;
+                    type string {
+                        pattern "Ethernet([1-3][0-9]{3}|[1-9][0-9]{2}|[1-9][0-9]|[0-9])|GLOBAL" {
+                            error-message "Invalid interface name";
+                            error-app-tag interface-name-invalid;
+                        }
+                    }
+                }
+
+                leaf action {
+                    default drop;
```

Comment:
As discussed, let's remove the "default" values for all the leaf attributes here. Backend default will be assumed for the configurations as per previous discussed in yang sub-group

Reply:
Removed default values for these leaf attributes.